cli/args/flags.rs

```diff
@@ -178,6 +178,7 @@ pub struct TestFlags {
   pub include: Vec<String>,
   pub filter: Option<String>,
   pub shuffle: Option<u64>,
+  pub parallel: bool,
```

Comment:
Why do we need this? Shouldn't we just check if concurrent_jobs > 1?

Reply:
I want it so that this output applies if `--parallel` is passed even if the amount of jobs set by the env is 1. The two options should perhaps be merged into
```rust
enum Concurrency {
  Sequential,
  Parallel(NonZeroUsize),
}
```

Reply:
Why?

Reply:
I don't agree because this is a display concern. This seems to adds more complexity for not much benefit.

Reply:
But the struct I suggested above is an accurate reflection of how the API works, either non-parallel or parallel with the number of threads deferred to `DENO_JOBS`. Rather than a configuration for the number of threads with a default. It's a larger structure than a single number but I wouldn't call that complexity...

Reply:
It's going to be super rare that someone has a system with 1 core and having extra code (complexity) to handle something cosmetic doesn't seem worth it.